.github/workflows/maketest.yml

```diff
@@ -1,41 +0,0 @@
-name: Makefile CI
-
-on:
-  push:
-    branches: [ "main" ]
-  pull_request:
-  merge_group:
-    branches: [ "main" ]
-
-jobs:
-  maketest:
-
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: ./libsql-sqlite3
-
-    steps:
-    - uses: actions/checkout@v3
-    
-    - name: get TCL
-      run: sudo apt-get install -y tcl8.6-dev
-
-    - name: Install Protoc
-      uses: arduino/setup-protoc@v2
-
-    - name: Install Wasmpack
-      uses: jetli/wasm-pack-action@v0.4.0
-      with:
-        version: 'latest'
-
-    - name: configure
-      run: ./configure
-      
-    - name: Run tests
-      run: make test
-
-    # Rust tests are run under maketestwasm
-
-    - name: Run API tests
-      run: make testlibsql
```

Comment:
why is this going as well?

Reply:
Yup as I said above, a lot of our make tests did a lot of the same so if we just do it with + wasm then it covers basically everything. This was after a chat with @psarna.